nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/parameter-contexts/ui/parameter-context-listing/parameter-context-table/parameter-context-table.component.ts

```diff
@@ -21,7 +21,7 @@ import { MatSortModule, Sort } from '@angular/material/sort';
 import { NiFiCommon } from '@nifi/shared';
 import { FlowConfiguration } from '../../../../../state/flow-configuration';
 import { CurrentUser } from '../../../../../state/current-user';
-import { ParameterContextEntity } from '../../../../../state/shared';
+import {ParameterContext, ParameterContextEntity} from '../../../../../state/shared';
```

Comment:
Please fix the formatting here. This can be accomplished by running:

`$ npx nx prettier:format`